loggo/loggo.py

```diff
@@ -371,15 +371,11 @@ def _represent_return_value(self, response, truncate=140):
         """
         Make a string representation of whatever a method returns
         """
-        representable = (int, float, str, list, set, dict, type(None), bool, tuple)
-        if isinstance(response, representable):
-            return '({})'.format(self._force_string_and_truncate(response, truncate, use_repr=True))
         # some custom handling for request response objects
-        content = getattr(response, 'content', False)
-        if content:
-            return '({})'.format(self._force_string_and_truncate(content.decode('utf-8'), truncate, use_repr=True))
-        # fallback, should not happen
-        return ''
+        if type(response) == 'requests.models.Response':
```

Comment:
Using this instead of `isinstance()` so we don't need to import the requests library. Not sure at all if this is better than just making the import though.